masonry/src/passes/event.rs

```diff
@@ -195,6 +195,23 @@ pub(crate) fn root_on_access_event(
         event,
         false,
         |widget, ctx, event| {
+            if event.target == ctx.widget_id() {
+                match event.action {
+                    accesskit::Action::Focus => {
+                        if ctx.is_in_focus_chain() && !ctx.is_disabled() && !ctx.is_focused() {
+                            ctx.request_focus();
+                        }
+                        return;
```

Comment:
There's an argument to be made that this return should be conditional only on `ctx.is_in_focus_chain()`, because that's when the corresponding implicit focus action was added.

However, I think that's probably YAGNI